Frameworks/CoreGraphics/CGContext.mm

```diff
@@ -1781,20 +1783,23 @@ static HRESULT _CreatePatternBrush(
 
     size_t bitsPerComponent = 8;
     size_t bytesPerRow = 4 * tileSize.size.width;
-    CGBitmapInfo bitmapInfo = kCGImageAlphaPremultipliedFirst | kCGBitmapByteOrder32Big;
+    CGBitmapInfo bitmapInfo = kCGImageAlphaPremultipliedFirst | kCGBitmapByteOrderDefault;
 
     woc::unique_cf<CGContextRef> patternContext{ CGBitmapContextCreate(
         nullptr, tileSize.size.width, tileSize.size.height, bitsPerComponent, bytesPerRow, colorspace.get(), bitmapInfo) };
     RETURN_HR_IF_NULL(E_UNEXPECTED, patternContext);
 
+    // Determine if this pattern is a colored pattern (the coloring is specified in the pattern callback) or if it is stencil pattern (the
+    // color is set outside and not within the pattern callback)
+    bool isColored = _CGPatternIsColored(pattern);
```

Comment:
it'll be duplicated in both stoke and fill.
I'll leave it here as the comments will also get duplicated.

---
In reply to: [100935286](https://github.com/Microsoft/WinObjC/pull/1952#discussion_r100935286) [](ancestors = 100935286)